services/catalog/src/simcore_service_catalog/db/repositories/services.py

```diff
@@ -216,18 +216,18 @@ async def update_service(
     ) -> ServiceMetaDataAtDB:
         # update the services_meta_data table
         async with self.db_engine.acquire() as conn:
-            row: RowProxy = await (
-                await conn.execute(
-                    # pylint: disable=no-value-for-parameter
-                    services_meta_data.update()
-                    .where(
-                        (services_meta_data.c.key == patched_service.key)
-                        & (services_meta_data.c.version == patched_service.version)
-                    )
-                    .values(**patched_service.dict(by_alias=True, exclude_unset=True))
-                    .returning(literal_column("*"))
+            result = await conn.execute(
```

Comment:
Is there a special reason why you separated results and row? or just for clarity? Just wondering whether there is a new limitation with the updated library

Reply:
no that is only for clarity